app/lib/slack_alerter.rb

```diff
@@ -0,0 +1,33 @@
+class SlackAlerter
+  SLACK_CHANNEL_EMAIL = 'apply-alerts-prod-aaaabbocbcpykkqticw7skurte@mojdt.slack.com'.freeze
```

Comment:
Should this be public or regenerated and set in the deployment settings on a per-environment basis?
Wouldn't this mean that a single channel gets _all_ alerts?!